build.sbt

```diff
@@ -17,6 +17,7 @@ scalacOptions := Seq(
   "-language:_",
   "-Xlog-reflective-calls")
 
+// SBT requires 2.10 for now (1/15/15).
```

Comment:
Yes.  The plugins library needs to compile for 2.10.  I get confused by this every time--I keep trying to bump it up to 2.11 but that's impossible.